### ISSUE TITLE
GVim DPI issues on Windows ARM64

### ISSUE THREAD

**Issue by wbpayne22902** [vim/vim#13455]
### Steps to reproduce

1. After finally compiling Vim and GVim on Windows ARM64 on my MacBook in Parallels there are some DPI issues with the toolbar that remain
2. The toolbar appears tiny for some reason.

![image](https://github.com/vim/vim/assets/107161609/cabfb7f3-62a0-47d6-ba06-b88b0afbf98f)

### Expected behaviour

Should the toolbar icons not be a bit bigger, is there anything I can do to make them bigger?

### Version of Vim

9.0.2081

### Environment

Operating System: Windows
No Terminal, this is a GVim issue.
Windows 11 Build 22635.2552
Visual Studio 2022

**Comment by k-takata** [comment]
Yes, currently only the button sizes are scaled but the icon images themself are not scaled because they will be blurred if scaled.
We may need to create larger icons.

**Comment by wbpayne22902** [comment]
Is there a way that I can configure larger but blurry icons?

**Comment by chrisbra** [comment]
No, unfortunately not. But it could be a nice side project. (However, except what is on the website or here in the repository, I don't have any original logos that could be used for creating new high-scale DPI logos, so most likely work must be re-done if the existing logos are not sufficient)
